### ISSUE TITLE
@apollo/react-testing installing old versions of ` jest-environment-jsdom` and `jest-environment-node`

### ISSUE THREAD

**Issue by AbreezaSaleem** [apollographql/react-apollo#3615]
I am using the latest version of jest which uses versions 24.9.0 of `jest-environment-jsdom` and `jest-environment-node`. But when i install `@apollo/react-testing`, it adds older versions (20.0.3) of `jest-environment-jsdom` and `jest-environment-node` in the root of `node_modules`, giving me the error: `TypeError: environment.teardown is not a function` when I run `npm test`. Can someone please assist me on this?

Basically its the same thing as discussed in https://stackoverflow.com/a/51812379/6051241 but instead of `css-modules` `@apollo/react-testing` seems to be causing the issue.
